Microsoft.Azure.CosmosRepository/src/Services/DefaultCosmosContainerSyncService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.CosmosRepository.Providers;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Azure.CosmosRepository.Services
+{
+    /// <inheritdoc/>
+    class DefaultCosmosContainerSyncService : ICosmosContainerSyncService
+    {
+        readonly ICosmosContainerService _containerService;
+
+        public DefaultCosmosContainerSyncService(ICosmosContainerService containerService) => _containerService = containerService;
+
+        public Task SyncAsync<TItem>() where TItem : IItem => _containerService.GetContainerAsync<TItem>(true);
```

Comment:
I guess one more suggestion, do you like this API? It's rather vague. What about `SynchronizeContainerPropertiesAsync` or `SyncContainerPropertiesAsync`?

Reply:
Yes, I think that is a fair comment, will amend this.